Glyssen/Dialogs/VoiceActorAssignmentDlg.cs

```diff
@@ -420,6 +420,9 @@ private void m_btnCancelMove_Click(object sender, EventArgs e)
 
 		private void m_characterGroupGrid_CellContentClick(object sender, DataGridViewCellEventArgs e)
 		{
+			// ignore clicks on the header row
+			if (e.RowIndex < 0) return;
```

Comment:
Won't this prevent the user from sorting?  I don't think we want to do that unless we have to.